posthog/api/action.py

```diff
@@ -73,6 +80,7 @@ class Meta:
 class ActionSerializer(serializers.HyperlinkedModelSerializer):
     steps = ActionStepSerializer(many=True, required=False)
     created_by = UserBasicSerializer(read_only=True)
+    is_calculating = serializers.SerializerMethodField()
```

Comment:
Is this a new thing? i.e. this isn't just a refactor?

Reply:
I just moved it from the ClickhouseActionViewset. It could be dead code but didn't bother refactoring too much yet